src/mock-doc/test/element.spec.ts

```diff
@@ -432,4 +432,18 @@ describe('element', () => {
       expect(input.list).toEqual(null);
     });
   });
+
+  describe('link', () => {
+    it('href', () => {
+      const elm = doc.createElement('a') as HTMLAnchorElement;
+      elm.href = 'http://stenciljs.com/path/to/page';
+      expect(elm.href).toBe('http://stenciljs.com/path/to/page');
+    });
+
+    it('pathname', () => {
+      const elm = doc.createElement('a') as HTMLAnchorElement;
```

Comment:
Since this is technically a `MockAnchorElement` returned by calling `createElement('a')`, can we type it as one?
```suggestion
      const elm: MockAnchorElement = doc.createElement('a');
```